scraper/src/fcc2zim/build.py

```diff
@@ -0,0 +1,138 @@
+import json
+import pathlib
+from collections import OrderedDict
+from datetime import datetime
+
+from zimscraperlib.zim import Creator
+
+from fcc2zim.constants import FCC_LANG_MAP, VERSION, Global
+
+logo_path = pathlib.Path(__file__).parent.joinpath("assets", "fcc_48.png")
+
+
+def build_curriculum_redirects(curriculum_dist_dir: pathlib.Path, language: str):
+    fcc_lang = FCC_LANG_MAP[language]
+    index_json_path = curriculum_dist_dir.joinpath("curriculum", fcc_lang, "index.json")
+    with open(index_json_path) as course_index_str:
+        superblock_dict = json.load(course_index_str)[fcc_lang]
+
+    redirects = []
+    for superblock in superblock_dict:
+        course_list = superblock_dict[superblock]
+        for course in course_list:
+            meta_json_path = pathlib.Path(
+                curriculum_dist_dir,
+                "curriculum",
+                fcc_lang,
+                superblock,
+                course,
+                "_meta.json",
+            )
+            challenges = json.loads(meta_json_path.read_text())["challenges"]
+            for challenge in challenges:
+                title = challenge["title"]
+                redirects.append(
+                    (f'{fcc_lang}/{superblock}/{course}/{challenge["slug"]}', title)
+                )
+
+    return OrderedDict(redirects).items()
+
+
+def build_command(
+    zimui_dist_dir: pathlib.Path,
+    out_dir: pathlib.Path,
+    language: str,
+    name: str,
+    title: str,
+    description: str,
+    long_description: str | None,
+    creator: str,
+    publisher: str,
+    curriculum_dist_dir: pathlib.Path,
+    zim_file: str | None,
+    *,
+    force: bool,
+):
+    Global.logger.info("Scraper: build phase starting")
+
+    period = datetime.today().strftime("%Y-%m")  # noqa: DTZ002
```

Comment:
I believe we've agreed on using a date for claritty

Reply:
A date like "2023-01-12" ?

Reply:
No I'm talking about using `datetime.date.today()` which returns a `date` and not a `datetime`. Clearer, doesn't require a TZ exception.
Using this as an opportunity to remind that we don't import `datetime` from `datetime` because it leads to confusion.